src/typeDefs/errors/mutationErrors.ts

```diff
@@ -0,0 +1,20 @@
+import { gql } from "graphql-tag";
+
+export const mutationErrrors = gql`
+  union AcceptAdminError = CurrentUserNotFound | GivenUserNotFound
+
+  interface UserNotFound {
```

Comment:
This interface serves no purpose. You should only have a base error interface like this:-
```graphql
interface Error {
  message: String!
}
``` 
All errors must have atleast the field named message. If needed they they can extend their types with more fields. Since errors are colocated with fields you'll mostly only need the message field to convey the error to the client.